manager/testdata/new-blueprints/copy-read-blueprint.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: Blueprint
+metadata:
+  name: copy-and-read
+  labels:
+spec:
+  cluster: theshire
+  appSelector: "app:demoapp"
+  appCluster: theshire
```

Comment:
Instead of appCluster and appSelector you can reuse https://github.com/mesh-for-data/mesh-for-data/blob/bedb8ecbbd215d733c2a4c6f2904efd8eeccb4df/manager/apis/app/v1alpha1/selector.go#L11

```
appSelector:
    clusterName: theshire
    workloadSelector:
      matchLabels:
        app: demoapp
```

Reply:
Sounds good! Integrated it in the new version.